requirements.txt

```diff
@@ -0,0 +1,8 @@
+coverage==5.0.3
```

Comment:
We don't want to have this file at all. Instead we are using `environment.yml` to have Conda handle dependencies. Please remove it.